cypress/plugins/index.js

```diff
@@ -6,6 +6,10 @@ module.exports = on => {
       resolve: {
         extensions: ['.ts', '.js']
       },
+      output: {
+        hotUpdateChunkFilename: '[id].[fullhash].hot-update.js',
+        hotUpdateMainFilename: '[runtime].[fullhash].hot-update.json',
+      },
```

Comment:
needed to prevent errors with the latest cypress. cf https://github.com/webpack/webpack/issues/6693#issuecomment-745688108

Reply:
I'd add such comments as code comments for future reader